### PR TITLE
Various fixes

### PR DESCRIPTION
**Pull request by Veykril**

### CODE REVIEW

File: src/expressions.rst [hunk 1]
```diff
@@ -412,6 +412,9 @@ location. The following :t:`[expression]s` are :t:`[place expression]s`:
 * :dp:`fls_ya05djl1d154`
   :t:`[Path expression]s` that resolve to a :t:`static` or a :t:`variable`.
 
+* :dp:`fls_gv4M0DE3OMwk`
+  A :t:`temporary`.
```
Comment (Dajamante) [inline src/expressions.rst:416]: Is there a description missing?

**Review by Dajamante — COMMENTED** [review]
Just the "A temporary".

File: src/expressions.rst (hunk 1 above)
Comment (Veykril) [reply src/expressions.rst:416]: description for what

**Review by Dajamante — APPROVED** [review]

**Comment by Dajamante** [comment]
bors merge

**Pull request merged**